### PR TITLE
dfdfdfdf

### PR DESCRIPTION
**Pull request by mohamedma872**
my first pull request

### CODE REVIEW

File: GithubPrinciples/GithubPrinciples/ViewController.swift [hunk 1]
```diff
@@ -16,6 +16,7 @@ class ViewController: UIViewController {
     }
     override func viewDidAppear(_ animated: Bool) {
          // Do any additional setup after loading the view.
+        let s = "sfsfsfsf"
```
Comment (Mahmoud-Bassuni) [inline GithubPrinciples/GithubPrinciples/ViewController.swift]: what is it ??

Comment (mohamedma872) [reply GithubPrinciples/GithubPrinciples/ViewController.swift]: this is commit for fixing issue

File: GithubPrinciples/GithubPrinciples/ViewController.swift [hunk 2]
```diff
@@ -16,6 +16,7 @@ class ViewController: UIViewController {
     }
     override func viewDidAppear(_ animated: Bool) {
```
Comment (Mahmoud-Bassuni) [inline GithubPrinciples/GithubPrinciples/ViewController.swift]:
```suggestion
    override func viewDidAppear(_ animated: Bool) {
```

Comment (Mahmoud-Bassuni) [inline GithubPrinciples/GithubPrinciples/ViewController.swift]:
```suggestion
    override func viewWillAppeaerr(_ animated: Bool) {
    
```

**Pull request merged**